### PR TITLE
Add reverse lookup for wallet address from twitter handle

### PR DESCRIPTION
**Pull request by kevinrodriguez-io**
This feature allows getting a reverse lookup for handle -> wallet

### CODE REVIEW

File: crates/graphql/src/schema/objects/profile.rs [hunk 1]
```diff
@@ -63,6 +67,21 @@ pub struct TwitterUserProfileResponse {
...
+            .filter(twitter_handle_name_services::twitter_handle.eq(&self.handle))
+            .load(&db_conn)
+            .context("Failed to load wallet address")?;
+        if result.len() == 0 {
+            return Ok(None);
+        }
+        let matching_item = result.get(0).unwrap();
+        let wallet_address = &matching_item.wallet_address;
+        Ok(Some(wallet_address.to_string()))
+    }
```
Comment (ray-kast) [inline crates/graphql/src/schema/objects/profile.rs:84]: I'm confused why wallet_address is stored on the Profile struct but also a computed field.

**Review by ray-kast — CHANGES_REQUESTED** [review]

File: crates/graphql/src/schema/objects/profile.rs (hunk 1 above)
Comment (kevinrodriguez-io) [reply crates/graphql/src/schema/objects/profile.rs:81]: You're right! Removed

Comment (kevinrodriguez-io) [reply crates/graphql/src/schema/objects/profile.rs:81]: You're right! Removed

**Review by ray-kast — APPROVED** [review]
Looks good, although some CI steps are still failing.

File: crates/graphql/src/schema/objects/profile.rs [hunk 2]
```diff
@@ -63,6 +64,21 @@ pub struct TwitterUserProfileResponse {
...
+            .filter(twitter_handle_name_services::twitter_handle.eq(&self.handle))
+            .load(&db_conn)
+            .context("Failed to load wallet address")?;
+        if result.is_empty() {
+            return Ok(None);
+        }
+        let matching_item = result.get(0).unwrap();
+        let wallet_address = &matching_item.wallet_address;
+        Ok(Some(wallet_address.to_string()))
+    }
```
Comment (kespinola) [inline crates/graphql/src/schema/objects/profile.rs:81]: This needs to be a data loader or we have n + 1.

Comment (kevinrodriguez-io) [reply crates/graphql/src/schema/objects/profile.rs:81]: @kespinola I'll create a ticket for this.

**Pull request merged**